tests/new_unit_tests/Commands/Connection/ConnectionCommandTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @file
+ *
+ * Contains Pantheon\Terminus\UnitTests\Commands\Connection\ConnectionCommandTest
+ */
+
+namespace Pantheon\Terminus\UnitTests\Commands\Connection;
+
+
+use ReflectionMethod;
+
+abstract class ConnectionCommandTest extends \PHPUnit_Framework_TestCase
```

Comment:
From my perspective, I feel the need to exercise the logic contained in a class. In this case the data structure (to enable the correct formatting) is part of the specification and the logic to perform is the domain of this class. Here I consider I am testing both user **and** developer concerns. The developer concerns requires me to access protected methods.

However, I am a 5/10 on this approach since the spec could be fully exercised in the behat run so I would be ok removing the protected method spying once that test was implemented. Keeping with this argument however would suggest to me that perhaps the entire `InfoCommandTest` should be obsolete.